www/source/blog/2018-02-21-Habitat-Helm.html.md

```diff
@@ -57,24 +43,29 @@ Once you've cloned core-plans locally, go to `core-plans/nginx` and enter the Ha
 
 * Enter the Habitat Studio within the nginx core plan with `hab studio enter`
 * Build the latest nginx package and stores it as a habitat artifact (“hart”) in the nginx plan’s `results` directory: `build`
-* Export the hab artifact to a Helm chart: `hab pkg export helm results/<yourfilename>.hart` This command will look very similar to `hab pkg export helm results/tdrew-nginx-1.11.10-20180221053714-x86_64-linux.hart`
+* Initialize the Helm client-side: `helm init -c`
+* Export the hab artifact to a Helm chart:
+
+`hab pkg export helm --download-deps results/<yourfilename>.hart`
+
+This command will look very similar to
+
+`hab pkg export helm --download-deps results/tdrew-nginx-1.11.10-20180221053714-x86_64-linux.hart`
+
 * Exit the Habitat Studio: `exit`
 
 You now have a Helm chart for core-plans/nginx in your core-plans/nginx/nginx-<version>-<datestamp>/ directory
 I suggest you pwd for the above ^^ so you can use it in another tab when you have started your cluster and want to know where your helm chart lives
 
 ### Start a cluster and deploy your helm chart!
 
-Here we are going to set up your minikube, and then deploy our helm chart onto that cluster. So, go back to your `habitat-operator` directory where you left your minikube cluster running and: 
+Here we are going to set up Helm in your Kubernetes cluster, and then deploy our Helm chart onto that cluster.
 
-* `kubectl create -f examples/rbac` This gives the Habitat Operator the permissions it needs on your RBAC-enabled cluster
-* `kubectl apply -f examples/rbac/habitat-operator.yml` Deploys the Habitat Operator in your minikube cluster 
 * `kubectl -n kube-system create sa tiller` Creates a service account for Tiller in the kube-system namespace
 * `kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceaccount=kube-system:tiller` Creates a ClusterRoleBinding for Tiller
 * `helm init --service-account tiller` Installs Tiller, specifying new service account 
 * `helm version` Check and make sure you have the right version of helm running on your client & server side, (as of printing, v 2.8.1) 
```

Comment:
Should this not be before the previous instruction?

Reply:
yeah